ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Context
```

Comment:
1. Add "Platform" and "Browser" sections.
2. "Possible fix" not needed.
3. "Process" rename to "Steps to reproduce".